confignode/src/test/java/org/apache/iotdb/confignode/service/thrift/ConfigNodeRPCServiceProcessorTest.java

```diff
@@ -1000,4 +1003,61 @@ private void cleanUserAndRole() throws TException {
       Assert.assertEquals(TSStatusCode.SUCCESS_STATUS.getStatusCode(), status.getCode());
     }
   }
+
+  @Test
+  public void testDeleteStorageGroup() throws TException {
```

Comment:
It seems that previous pr changes has been removed... This function is the same as deleteStorageGroupTest(). I just change the name. Test class uses ....Test format and test function uses test... format should be more normalized.

Reply:
Ok, get it! I will have a check.